app/src/main/res/values-de-rDE/strings.xml

```diff
@@ -0,0 +1,185 @@
+<resources>
```

Comment:
In this file a lot of the strings have been translated but the boolean that if they should be translated is set to false. Cirrus does not like that!